packages/lesswrong/client/autoRefresh.ts

```diff
@@ -37,9 +35,23 @@ function connectWebsocket() {
   connectedWebsocket.addEventListener("open", (event: OpenEvent) => {
   });
   connectedWebsocket.addEventListener("error", (event: CloseEvent) => {
+    disconnectWebsocket();
+  });
+  connectedWebsocket.addEventListener("close", (event: CloseEvent) => {
+    disconnectWebsocket();
   });
 }
 
+let reloadStarted = false;
+function refreshForNewVersion(newVersionTimestamp: string) {
+  if (!reloadStarted) {
+    reloadStarted = true;
```

Comment:
Do we not need to reset this to `false`?

Reply:
Nope. Once a refresh is started, either the page is refreshed (in which case nothing else matters), or the refresh is rejected by the user (eg by clicking Cancel on an unsaved-changes-warning popup), in which case we shouldn't try again.